README.rst

```diff
@@ -24,6 +24,14 @@ Note: the ``containers`` subcommand requires ``docker`` to already be installed
 on your machine. Please see the `docker installation instructions <http://docs.docker.com/index.html>`_ for platform
 specific information.
 
+autocomplete
+^^^^^^^^^^^^
+
+To enable tab autocomplete, please install `argcomplete <https://github.com/kislyuk/argcomplete>`_ using ``pip install autocomplete``.
```

Comment:
Hmm, I don't think there's much harm to adding `argcomplete` to `setup.py` as a dependency and leaving it up to the user to run the `register-python-argcomplete` command or setting up global autocompletion.